docs/tasks/20260411_0824_支出先レンジ表示オプション設計.md

```
@@ -0,0 +1,98 @@
+
+#### 1. `__agg-recipient` ノードを非表示
+
+- 集約支出先ノード（`その他の支出先`）を出力しない
+- それに向かうエッジ（`project-spending → __agg-recipient`、`__agg-project-spending → __agg-recipient`）も出力しない
+
+#### 2. ウィンドウ内支出先に接続のない事業を非表示
+
+通常モードの `effectivelyHiddenIds` はウィンドウ・テール両方への流出がゼロの事業を隠す。
+OFF 時はテールへの流出があっても **ウィンドウへの流出がゼロの事業を隠す**。
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)